scripts/download-sponsors.js

```diff
@@ -6,60 +6,189 @@ const fs = require("fs");
 
 const graphqlEndpoint = "https://api.opencollective.com/graphql/v2";
 
-const graphqlQuery = `{
+// all from webpack: https://github.com/webpack/webpack.js.org/blob/master/src/utilities/fetch-supporters.js
+const { promisify } = require("util");
+const asyncWriteFile = promisify(fs.writeFile);
+const REQUIRED_KEYS = ["totalDonations", "slug", "name"];
+const sponsorsFile2 = `${__dirname}/../website/data/sponsors.json`;
```

Comment:
Why `sponsorsFile2` and not `sponsorsFile`?

Reply:
carry over when I was testing with the old one 😛